shell/platform/common/client_wrapper/include/flutter/basic_message_channel.h

```diff
@@ -101,6 +102,41 @@ class BasicMessageChannel {
     messenger_->SetMessageHandler(name_, std::move(binary_handler));
   }
 
+  // Adjusts the number of messages that will get buffered when sending messages
+  // to channels that aren't fully set up yet. For example, the engine isn't
+  // running yet or the channel's message handler isn't set up on the Dart side
+  // yet.
+  //
+  // |new_size] is an int because the deserialization logic handles only 32 bits
+  // values, see
+  // https://github.com/flutter/engine/blob/93e8901490e78c7ba7e319cce4470d9c6478c6dc/lib/ui/channel_buffers.dart#L495.
```

Comment:
This seems like a note intended for us, rather than client-facing, so it should be in the implementation code (e.g., on line 114).